lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -86,8 +86,15 @@ def index(self, trans, **kwds):
     @expose_api_anonymous_and_sessionless
     def show(self, trans, id, **kwd):
         """
-        GET /api/tools/{tool_id}
-        Returns tool information, including parameters and inputs.
+        GET /api/tools/{tool_id} 
```

Comment:
There's a whitespace after `/api/tools/{tool_id}`, you need to remove this for the linting to not raise an error.
Also this should be `/api/tools/{tool_id}/show`